test/cql-pytest/test_type_time.py

```diff
@@ -49,3 +57,67 @@ def test_type_time_from_string_prepared(cql, table1):
     for t in ["08:12:54", "08:12:54.123", "08:12:54.123456", "08:12:54.123456789"]:
         cql.execute(stmt, [p, t])
         assert list(cql.execute(f"SELECT t from {table1} where p = {p}")) == [(Time(t),)]
+
+# According to the CQL binary protocol spec, `time` values must be between 0 and 24 hours.
+# Test that Scylla rejects integer values outside of this range
+# https://github.com/apache/cassandra/blob/f5df4b219e063cb24b9cc0c22b6e614506b8d903/doc/native_protocol_v4.spec#L941
+# An 8 byte two's complement long representing nanoseconds since midnight.
+# Valid values are in the range 0 to 86399999999999
```

Comment:
Sadly, https://cassandra.apache.org/doc/latest/cassandra/cql/types.html#times says something different: "64-bit signed integers representing the number of nanoseconds since midnight." - there is no mention of the upper limit, and the word "signed" seems to suggest negative numbers are also allowed.
We'll need to decide which is the correct behavior, and why. Maybe it's worth reporting this discrepency to Cassandra and see what they think.